src/App.tsx

```diff
@@ -14,11 +14,76 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+type State = {
+  selectedGood: string[],
+};
+
+class App extends React.Component<{}, State> {
+  state = {
+    selectedGood: ['Jam'],
+  };
+
+  selected = (goods: string[]) => {
+    if (goods.length === 1) {
+      return ` ${goods[0]}`;
+    }
+
+    return goods.map(item => (goods.indexOf(item) + 1 !== goods.length
```

Comment:
You can use index from arguments of callback function
```suggestion
    return goods.map((item, index) => index + 1 !== goods.length
```

Reply:
right, I forgot